securedrop/source_app/main.py

```diff
@@ -292,9 +292,9 @@ def submit(logged_in_source: SourceUser) -> werkzeug.Response:
             elif msg and not fh:
                 html_contents = gettext("Thanks! We received your message.")
             elif fh and not msg:
-                html_contents = gettext("Thanks! We received your document.")
+                html_contents = gettext("Thanks! We received your file.")
             else:
-                html_contents = gettext("Thanks! We received your message and document.")
+                html_contents = gettext("Thanks! We received your submission.")
```

Comment:
It sounds a little clumsy, but I would err on the side of "your file and message", just so users are explicitly informed that both were received.